README.md

```diff
@@ -8,6 +8,34 @@ zoltar
 
 Common library for serving TensorFlow and XGBoost models in production.
 
+Zoltar is a library that helps mount predictive machine learning models in [Apollo](https://github.com/spotify/apollo) and other Java services. It provides several key objects which allow your service to load a serialized model, featurize input data, submit to the model for prediction, and serve that prediction. Note that these are all abstract objects which you will have to implement for your particular use case. Below is a quick overview of these abstractions which can be found in [zoltar-core](https://github.com/spotify/zoltar/tree/6390c056d9e3d033bdaf1c2dedd4901e309571ec/zoltar-core).
```

Comment:
* `key objects` -> `key abstractions`. 
 * would remove `Note that these are all abstract objects which you will have to implement for your particular use case.`